From 0ef4d03899626852d39b783e19cf72420f51d4b2 Mon Sep 17 00:00:00 2001
From: Mark Geisert <mark@maxrnd.com>
Date: Thu, 21 Apr 2022 22:36:33 -0700
Subject: [PATCH 062/N] Cygwin: Fix "0x0x" in gmondump and ssp man pages

A recent patch fixed gmondump to stop printing "0x0x" as an address
prefix.  It turns out the Cygwin User's Guide and the gmondump and
ssp man pages (all from utils.xml) have examples of the same error.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/release/3.3.5 | 2 +-
 winsup/doc/utils.xml        | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/winsup/cygwin/release/3.3.5 b/winsup/cygwin/release/3.3.5
index f0a8340..049d19b 100644
--- a/winsup/cygwin/release/3.3.5
+++ b/winsup/cygwin/release/3.3.5
@@ -42,7 +42,7 @@ Bug Fixes
   Addresses: https://cygwin.com/pipermail/cygwin/2022-March/251022.html
 
 - Fix a formatting problem in gmondump where all displayed addresses are
-  mistakenly prefixed with "0x0x".
+  mistakenly prefixed with "0x0x". Fix man pages for gmondump and ssp.
 
 - Fix crash on pty master close in Windows 7.
   Addresses: https://cygwin.com/pipermail/cygwin/2022-March/251162.html
diff --git a/winsup/doc/utils.xml b/winsup/doc/utils.xml
index 0b9e385..8959880 100644
--- a/winsup/doc/utils.xml
+++ b/winsup/doc/utils.xml
@@ -846,7 +846,7 @@ line separates the ACLs for each file.
 <screen>
 $ gmondump gmon.out.21900.zstd.exe
 file gmon.out.21900.zstd.exe, gmon version 0x51879, sample rate 100
-  address range 0x0x100401000..0x0x1004cc668
+  address range 0x100401000..0x1004cc668
   numbuckets 208282, hitbuckets 1199, hitcount 12124, numrawarcs 0
 </screen>
     </refsect1>
@@ -2951,7 +2951,7 @@ Idx Name          Size      VMA       LMA       File off  Algn
       section and the VMA of the section after it (sections are usually
       contiguous; you can also add the Size to the VMA to get the end address).
       In this case, the VMA is 0x61001000 and the ending address is either
-      0x61080000 (start of .data method) or 0x0x6107fa00 (VMA+Size method). </para>
+      0x61080000 (start of .data method) or 0x6107fa00 (VMA+Size method). </para>
 
     <para> There are two basic ways to use SSP - either profiling a whole
       program, or selectively profiling parts of the program. </para>
-- 
2.9.0

